src/triggers.js

```diff
@@ -31,7 +37,7 @@ const baseStore = function() {
 };
 
 function validateClassNameForTriggers(className, type) {
-  const restrictedClassNames = ['_Session'];
+  const restrictedClassNames = [];
   if (restrictedClassNames.indexOf(className) != -1) {
```

Comment:
What’s the point of keeping that? Perphas use ‘readOnlyClassNames’ instead

Reply:
no point, will remove it